### ISSUE TITLE
Custom arguments passed are no longer being passed to python file being debugged

### ISSUE THREAD

**Issue by DonJayamanne** [Microsoft/ptvsd#302]
Note: This used to work
* Could be related to #302 and #299

* Use VSC to debug a program
* Add custom arguments in `launch.json` by adding items into the `args` (array) property
* Debug a python file with the following contents:
```python
import sys
print(sys.argv)
```
* With the per-release version of PTVSD, custom args are passed through to the python file,
* With the master branch of PTVSD, none of the args are passed to the python file

* launch.json
You could either pip install PTVSD into your python interpreter or just add the path to `PYTHONPATH` as shown below:
```json
        {
            "name": "Run file",
            "type": "pythonExperimental",
            "request": "launch",
            "program": "${file}",
            "args": [ "one", "two", "three" ],
            "env": {
                "PYTHONPATH": "<Path to PTVSD>"
            }
        },
```

**Comment by ericsnowcurrently** [comment]
@DonJayamanne Please verify that this is fixed now that #306 has been merged.

**Comment by DonJayamanne** [comment]
Fixed
